src/main/java/com/amazon/jenkins/ec2fleet/EC2RetentionStrategy.java

```diff
@@ -18,6 +19,7 @@
 public class EC2RetentionStrategy extends RetentionStrategy<EC2FleetNodeComputer> implements ExecutorListener {
     private static final Logger LOGGER = Logger.getLogger(EC2RetentionStrategy.class.getName());
     private static final int RE_CHECK_IN_A_MINUTE = 1;
+    private static final long ORPHAN_TIME_LIMIT = TimeUnit.MINUTES.toMillis(1);
```

Comment:
We should probably surface this config to the user at some point? Might be a good idea to do it after we get some feedback from the user.

Reply:
Yes the value and its visibility is just for testing until we decide on a proper design. I think 1 minute is not a reasonable timer, but used this value for testing purposes. Will update it when the changes are ready.